### ISSUE TITLE
fails with -v, does not work without -v either.

### ISSUE THREAD

**Issue by vzaliva** [joh/when-changed#92]
```when-changed -v -s paper.tex paper.bib Makefile -c "make clean || make paper.pdf || beep"
When 'paper.tex', 'paper.bib' or 'Makefile' changes, run 'make clean || make paper.pdf || beep'
Traceback (most recent call last):
  File "/home/lord/.local/bin/when-changed", line 8, in <module>
    sys.exit(main())
  File "/home/lord/.local/lib/python3.8/site-packages/whenchanged/whenchanged.py", line 244, in main
    wc.run()
  File "/home/lord/.local/lib/python3.8/site-packages/whenchanged/whenchanged.py", line 163, in run
    self.run_command('/dev/null')
  File "/home/lord/.local/lib/python3.8/site-packages/whenchanged/whenchanged.py", line 96, in run_command
    print_message = "'" + thefile + "' " + re.sub(r'^[^_]+_', '', self.get_envvar('event'))
  File "/home/lord/.local/lib/python3.8/site-packages/whenchanged/whenchanged.py", line 159, in get_envvar
    return self.process_env['WHEN_CHANGED_' + name.upper()]
KeyError: 'WHEN_CHANGED_EVENT'
make: *** [Makefile:37: watch] Error 1
```

Without `-v` no error is shown, but command is not executed when files change.

**Comment by joh** [comment]
I think this was fixed in #91, can you please test again?

**Comment by vzaliva** [comment]
Sorry, I am no longer using this program. Since it was broken, I've switched to another one. 
Glad you fixed it. Hopefully, it will help other users.
